appveyor.yml

```diff
@@ -1,21 +1,24 @@
 cache:
   - node_modules
+  - "%LOCALAPPDATA%\\Yarn"
+
+init:
+  - git config --global core.autocrlf true
```

Comment:
-1 for that. We need to make sure that encore works *without* altering the source code (because normal usage will not do such conversion)